llvm/lib/MC/MCDwarf.cpp

```diff
@@ -1518,6 +1538,8 @@ void FrameEmitterImpl::emitCFIInstruction(const MCCFIInstruction &Instr) {
     }
     return;
   }
+  case MCCFIInstruction::OpLLVMRegOffset:
+    llvm_unreachable("Should emit llvm_reg_offset as escape");
```

Comment:
It looks like all other instructions in this function emit something to the streamer. Why don't we emit something to the streamer here? Can you help me understand what it means to emit it as an escape? Where does this occur? Are there any other MCCFIInstructions that do something similar?

If you are expected to escape this in the CFIInserter, I think I'd like to see that code included in this patch.